service/src/models/archaeologist.ts

```diff
@@ -153,29 +153,34 @@ export class Archaeologist {
   }
 
   async publishEnvConfig(connection) {
```

Comment:
We may want to rename this function to something like `streamPublicKeyToEmbalmer`, or something more explicit, since we ripped out the env config data being sent